docs/collaboration.md

```diff
@@ -0,0 +1,66 @@
+# Collaborative Publishing with Posit Publisher
+
+[Posit Publisher](https://github.com/posit-dev/publisher)—the extension to
+publish content to Posit Connect using
+[Visual Studio Code](https://code.visualstudio.com/) and
+[Positron](https://positron.posit.co/)—enables seemless collaboration. This
+workflow guide shows how to collaborate with others on a piece of content.
+
+## Initial setup
+
+To ensure that others can collaborate without issue the deployment and
+configuration files should be available.
+
+That means including them in the repository for the project or including them
+in the `files` attribute in the configuration so they are available in the
+content bundle.
+
+When Posit Publisher creates a configuration it has the deployment and
+configuration files included for deployment.
+
+The inclusions are in the `files` list and look like:
+
+- `/.posit/publish/deployments/deployment-58A3.toml` for the deployment file
+- `/.posit/publish/content-title-061E.toml` for the configuration file
+
+The inclusion of the deployment and configuration files allows others to easily
+collaborate when they use the Source Version content bundle as a starting point.
+
+## As a collaborator
+
+### Start from a repository
+
+Clone the repo
+
+### Start from a content bundle
+
+Download the Source Version content bundle. If .posit files are included.
+
+They are included by default
```

Comment:
not sure about the period at the end of line 37 vs this next line of being included by default? 

Were you trying to address the fact that if publisher is used to deploy the project, the `.posit` files are included by default? 

That brings up another scenario we should address, when publisher was NOT used to deploy a project, what are the steps required to get collaboration to occur.

Reply:
This is just unstructured brain dumping on my part. The intention here was to note that if the content bundle doesn't include the `.posit` files I will need to write something.

> when publisher was NOT used to deploy a project, what are the steps required to get collaboration to occur.

This is exactly what I mean to include in the final document. It will probably be a two part suggestion of:
- setup the `.posit` files by doing the initial setup above
- re-publish

It will take some refining to avoid repeating too much information. I'm not sure if the "As a collaborator" and "Initial Setup" sections are the most helpful yet.